Frameworks/UIKit/UIViewController.mm

```diff
@@ -1254,40 +1240,64 @@ - (void)dismissModalViewControllerAnimated:(BOOL)animated {
     [self dismissViewControllerAnimated:animated completion:nil];
 }
 
-- (void)_notifyDidDisappearAnimated:(UIView*)view {
+- (void)_notifyDidDisappear:(UIView*)view animated:(BOOL)animated {
     LLTREE_FOREACH(subview, view->priv) {
-        [self _notifyDidDisappearAnimated:subview->self];
+        [self _notifyDidDisappear:subview->self animated:animated];
     }
 
     UIViewController* controller = [UIViewController controllerForView:view];
 
     if (controller != nil) {
-        [controller _notifyViewDidDisappear:TRUE];
+        [controller _notifyViewDidDisappear:animated];
     }
 }
 
-- (void)_dismissTransitionStopped:(id)anim finished:(BOOL)finished {
-    UIView* view = [priv->_dismissController view];
-    [[view layer] removeAnimationForKey:@"ModalDismiss"];
+- (void)_unlinkPresentedController {
```

Comment:
rename of method made the diff worse but |_unlinkPresentedController| is the same method as the previous |_childDismissCleanup|.